src/module/iotjs_module_i2c.cpp

```diff
@@ -49,6 +48,19 @@ I2c* I2c::GetInstance() {
 }
 
 
+void GetI2cArray(JObject* jarray, I2cReqData* req_data) {
+  // FIXME: Need to implement a function to get array info from JObject Array.
+  req_data->buf_len = jarray->GetProperty("length").GetNumber();
```

Comment:
There is function ``IsArray`` to check it is array object or not. I checked the type of JObject before calling this function.